### PR TITLE
Add list of known users

### PR DESCRIPTION
**Pull request by evenh**
I started on a list of known users of db-scheduler. What do you @kagkarlsson?

### CODE REVIEW

**Comment by evenh** [comment]
I see that Vy is also mentioned in https://github.com/kagkarlsson/db-scheduler/issues/86#issuecomment-581173729. We could also maybe add Nordea? cc @cash1981

File: README.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ See also [why not Quartz?](#why-db-scheduler-when-there-is-quartz)
 * **Simple**.
 * **Minimal dependencies**. (slf4j)
 
+## Known users
```
Comment (kagkarlsson) [inline README.md:19]: "Who uses db-scheduler" ?

**Comment by kagkarlsson** [comment]
You can add 
```
Vy Group – one of the largest transport groups in the Nordic countries.
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ See also [why not Quartz?](#why-db-scheduler-when-there-is-quartz)
 * **Simple**.
 * **Minimal dependencies**. (slf4j)
 
+## Who uses db-scheduler?
+
+db-scheduler is used by many individuals and businesses. Feel free to open a PR to add your company to the list.
```
Comment (kagkarlsson) [inline README.md:21]: List of organizations known to be running db-scheduler in production:

File: README.md [hunk 3]
```diff
@@ -16,6 +16,14 @@ See also [why not Quartz?](#why-db-scheduler-when-there-is-quartz)
...
+
+| Company                         | Description                                      |
+|---------------------------------|--------------------------------------------------|
+| [Digipost](https://digipost.no) | A Norwegian provider of digital secure mailboxes |
+
```
Comment (kagkarlsson) [inline README.md:26]: Feel free to open a PR to add your organization to the list.

**Pull request merged**